<ui version="4.0" >
 <class>chatWin</class>
 <widget class="QWidget" name="chatWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Contact's jid</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTextEdit" name="discutionText" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="undoRedoEnabled" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QLineEdit" name="messageLine" />
     </item>
     <item>
      <widget class="QPushButton" name="sendFileBtn" >
       <property name="text" >
        <string>Send file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendBtn" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
